nix/pkgs/images/default.nix

```diff
@@ -110,14 +113,44 @@ in
     copyToRoot = [ busybox io-engine ];
     config = { Entrypoint = [ "/bin/io-engine-client" ]; };
   });
-
-  mayastor-fio-spdk = dockerTools.buildImage (clientImageProps // {
-    name = "${repo-org}/${img_prefix}-fio-spdk";
-    copyToRoot = clientImageProps.copyToRoot ++ [ tini fio_wrapper ];
+  mayastor-io-engine-client-dev = dockerTools.buildImage (ioEngineImageProps // {
+    name = "${repo-org}/${img_prefix}-io-engine-client";
+    copyToRoot = [ busybox io-engine-dev ];
+    config = { Entrypoint = [ "/bin/io-engine-client" ]; };
   });
 
   mayastor-casperf = dockerTools.buildImage (clientImageProps // {
     name = "${repo-org}/${img_prefix}-casperf";
     copyToRoot = clientImageProps.copyToRoot ++ [ tini casperf ];
   });
+  mayastor-casperf-dev = dockerTools.buildImage (clientImageProps // {
+    name = "${repo-org}/${img_prefix}-casperf";
+    copyToRoot = clientImageProps.copyToRoot ++ [ tini casperf ];
+  });
+
+  mayastor-fio-spdk = dockerTools.buildImage (clientImageProps // {
+    name = "${repo-org}/${img_prefix}-fio-spdk";
+    copyToRoot = clientImageProps.copyToRoot ++ [ tini fio_wrapper ];
+  });
+in
+{
+  # keep for back-compat, in case it's necessary
+  inherit mayastor-io-engine mayastor-io-engine-dev mayastor-casperf mayastor-fio-spdk;
+
+  release = {
+    mayastor = {
+      io-engine = mayastor-io-engine;
+      io-engine-client = mayastor-io-engine-client;
+      casperf = mayastor-casperf;
```

Comment:
do we really need casperf for release build?

Reply:
Yes, in fact you might ask the opposite, do we need it for debug builds :)
Sometimes it's useful to test it in debug because the debug build is quicker